test/common/run-tests

```diff
@@ -548,6 +550,8 @@ def main():
                         help='Update the occurrence of naughties on cockpit-project/bots')
     parser.add_argument('--no-retry-fail', action='store_true',
                         help="Don't retry failed tests")
+    parser.add_argument('--amplify', type=int, default=1,
+                        help="Number of times to repeatedly run the given tests")
```

Comment:
Ah, so this is for the case where you want to run *all* tests multiple times. But that's not practical for our default CI case where all tests run, it would take way too long. I thought this would take a test name and only repeat that one. Of course it's always possible to just run a single test, but I often find that this changes the outcome -- you need the noise of high parallelism to reproduce races (and this only makes sense for race conditions)